Improve documentation (#2280)
tailwindlabs/tailwindcss.com: src/docs/preflight.mdx

Source detection affects generated utilities so `source(…)` should be added to the `utilities.css` import:

```css
/* [!code filename:CSS] */
@layer theme, base, components, utilities;

@import "tailwindcss/theme.css" layer(theme);
@import "tailwindcss/utilities.css" layer(utilities);  /* [!code --] */
@import "tailwindcss/utilities.css" layer(utilities) source(none);  /* [!code ++] */
```

The same goes for `important` which affects generated utilities:

```css
/* [!code filename:CSS] */
@layer theme, base, components, utilities;

@import "tailwindcss/theme.css" layer(theme);
@import "tailwindcss/utilities.css" layer(utilities);  /* [!code --] */
@import "tailwindcss/utilities.css" layer(utilities) important;  /* [!code ++] */
```

Likewise, `theme(static)`, `theme(inline)`, etc… affect generated theme variables and should be placed on the `theme.css` import:

```css
/* [!code filename:CSS] */
@layer theme, base, components, utilities;

@import "tailwindcss/theme.css" layer(theme);  /* [!code --] */
@import "tailwindcss/theme.css" layer(theme) theme(static);  /* [!code ++] */
@import "tailwindcss/utilities.css" layer(utilities);
```

> Using a prefix, e.g. `prefix(tw)`, affects *both* so it should go on both imports:

Reviewer: ```suggestion
Finally, using a prefix with `prefix(tw)` affects the utilities and variables, so it should go on both imports:
```
